### ISSUE TITLE
Android build failure with v2.2.0 - RNDatePickerDialogFragment.java:105: error: missing return statement

### ISSUE THREAD

**Issue by nadav2051** [react-native-community/react-native-datetimepicker#104]
# Bug

Android fails to build version 2.2.0.

`> Task :@react-native-community_datetimepicker:compileReleaseJavaWithJavac FAILED
/data/workspace/AndroidReactBuildNew/node_modules/@react-native-community/datetimepicker/android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java:105: error: missing return statement
  }
  ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
`

## Environment info

React native info output:

```bash
  IDEs:
    Android Studio: 3.4 AI-183.6156.11.34.5522156
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    @react-native-community/cli: ^1.9.5 => 1.12.0 
    react: 16.8.6 => 16.8.6 
    react-native: 0.60.4 => 0.60.4

```

Library version: 2.2.0

## Steps To Reproduce

Try to build.

Describe what you expected to happen:

App to build successfully.

**Comment by chiay** [comment]
Getting the same error here.

**Comment by Unforgiven-wanda** [comment]
Same error.

**Comment by kuwaitbinary** [comment]
I am returning null as temporary fix **not ideal**  

```
      DatePickerDialog dialog = new RNDismissableDatePickerDialog(activityContext, onDateSetListener, year, month, day);
      switch (display) {
        case CALENDAR:
          dialog.getDatePicker().setCalendarViewShown(true);
          dialog.getDatePicker().setSpinnersShown(false);
          break;
        case SPINNER:
          dialog.getDatePicker().setCalendarViewShown(false);
          break;
      }
      return dialog;
    }
    return null;
  }

```

<img width="1161" alt="Screen Shot 2020-02-03 at 11 54 52 AM" src="https://user-images.githubusercontent.com/1690779/73638883-4a9e5180-467c-11ea-86fa-ee3d054df7ec.png">

**Comment by cristianoccazinsp** [comment]
How did a compile a error get past a release... Same issue here, rolling back to 2.1 until then.

**Comment by yesterday24** [comment]
Same error. Currently fixed it with installing older version (2.1.0).
